recipes/lenapy/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "lenapy" %}
+{% set version = "0.9" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/CNES/lenapy/archive/refs/heads/main.zip
```

```suggestion
  url: https://github.com/CNES/lenapy/archive/refs/tags/V{{ version }}.tar.gz
```
This needs to be built from a tagged release, not the head commit.  If it cannot be built from the latest tagged release, I'm afraid we can't accept the recipe.

---

Sorry, I wasn't aware of this constraint, but it seems perfectly justified. I updated the recipe